### PR TITLE
update /dev/md0 to /dev/mdx

### LINKED ISSUE #1453: Terraform deploy on gke failed
## Bug Report

**What version of Kubernetes are you using?**

1.15.4
**What version of TiDB Operator are you using?**

v3.0.5
**What storage classes exist in the Kubernetes cluster and what are used for PD/TiKV pods?**

```
# kubectl --kubeconfig=credentials/kubeconfig_gkedan get sc
NAME                 PROVISIONER                    AGE
local-storage        kubernetes.io/no-provisioner   3h38m
pd-ssd               kubernetes.io/gce-pd           3h38m
standard (default)   kubernetes.io/gce-pd           3h41m
```
TiKV uses `local-storage`.
**What's the status of the TiDB cluster pods?**

```
# kubectl --kubeconfig=credentials/kubeconfig_gkedan get pod -n tidb-cluster   
NAME                                      READY   STATUS    RESTARTS   AGE
tidb-cluster-discovery-746c9dd846-s4qnk   1/1     Running   0          3h13m
tidb-cluster-monitor-66bdc59c-rvlq4       3/3     Running   0          3h13m
tidb-cluster-pd-0                         1/1     Running   0          10m
tidb-cluster-pd-1                         1/1     Running   1          3h13m
tidb-cluster-pd-2                         1/1     Running   0          3h13m
tidb-cluster-tikv-0                       0/1     Pending   0          19s
tidb-cluster-tikv-1                       0/1     Pending   0          18s
tidb-cluster-tikv-2                       0/1     Pending   0          18s
```
**What did you do?**

Deploy TiDB cluster on GKE with terraform.
**What did you expect to see?**
TiDB Cluster is deployed successfully.
**What did you see instead?**
TiKV Pods are pending.

### PR DESCRIPTION
**Pull request by DanielZhangQD**
### What problem does this PR solve? 
fix #1453 
### What is changed and how does it work?
/dev/md0 now exists by default on GKE nodes even without local ssd attached, so we cannot check /dev/md0 in the startup script of local-volume-provisioner to determine whether RAID has been applied.
### Check List 

Tests 

 - Manual test

Code changes

 - Has terraform script change

Related changes

 - Need to cherry-pick to release-1.0, release-1.1

### Does this PR introduce a user-facing change?:
 
 ```release-note
Fix the GKE deployment failure with terraform.
 ```

### CODE REVIEW

File: manifests/gke/local-ssd-provision/local-ssd-provision.yaml [hunk 1]
```diff
@@ -99,7 +99,7 @@ spec:
                 done
 
                 devs=$(echo "$ssd_mounts" | awk '{print $2}')
-                raid_dev=/dev/md0
+                raid_dev=/dev/mdx
```
Comment (aylei) [inline manifests/gke/local-ssd-provision/local-ssd-provision.yaml]: Is `/dev/mdx` the general case? maybe we should consider not using fixed path and list the devices first

**Comment by cofyc** [comment]
what is `/dev/mdx` for? can you link some docs about it.

**Comment by gregwebs** [comment]
> what is /dev/mdx for? can you link some docs about it.

This is the device for raiding of the disks in the script.

File: manifests/gke/local-ssd-provision/local-ssd-provision.yaml (hunk 1 above)
Comment (gregwebs) [reply manifests/gke/local-ssd-provision/local-ssd-provision.yaml]: But we need to know what device we have chosen if this script is ran a 2nd time (avoiding data loss is the most important thing). We could write this to /var/raid_mount_point to

**Comment by DanielZhangQD** [comment]
The startup script in release-1.0 does not include mount operations with mdadm, so no need to cherry-pick to release-1.0.

**Pull request merged**